src/main/java/net/datafaker/Nigeria.java

```diff
@@ -0,0 +1,37 @@
+package net.datafaker;
+
+import java.nio.file.Paths;
+import java.util.Locale;
+
+public class Nigeria {
+    private static final String KEY = "nigeria";
+    private final Faker faker;
+
+    protected Nigeria(Faker faker) {
+        this.faker = faker;
+        faker.fakeValuesService().addPath(Locale.ENGLISH, Paths.get("target/classes/en/nigeria.yml"));
```

Comment:
Wrong location

Reply:
Enlighten me please, wrong location for the class, or the yml file path?